runtime/compiler/control/J9Options.cpp

```diff
@@ -1752,53 +1732,62 @@ J9::Options::fePreProcess(void * base)
          }
       else
          {
-            UDATA largePageSize = 0;
-            UDATA largePageFlags = 0;
-            // -Xlp<size>, attempt to use specified page size
-            if (lpSize > 0)
+
+         UDATA largePageSize = 0;
+         UDATA largePageFlags = 0;
+         int32_t lpArgIndex;
+         UDATA lpSize = 0;
+
+         // -Xlp<size>, attempt to use specified page size
+         if ((lpArgIndex=FIND_ARG_IN_VMARGS(EXACT_MEMORY_MATCH, "-Xlp", 0)) >= 0)
+            {
+            GET_MEMORY_VALUE(lpArgIndex, "-Xlp", lpSize);
+            }
+
```

Comment:
@gita-omr Everything further, if you take a look at the current version of this file you'll see that a portion of this cod is incorrectly indented. So, I've simply fixed the spacing.

Reply:
For complicated diffs, I usually pull in the PR changes locally and use the diff tool on my system. To do this:

```
# Set PR number
$prnum = 7563

# Fetch PR
git fetch upstream pull/$prnum/head:pr$prnum

# Checkout PR
git checkout pr$prnum
```
Now that I have a local branch:
```
$ git log --graph --oneline -n5
* bba5a43eb3 (HEAD -> pr7563) Remove Unused Options
*   f740577d3a Merge pull request #7546 from doomerxe/FixCacheUniqueID
|\  
| * 2b45511725 Memset the shared classes CacheUniqueID
* |   d81b55019f J9 PrivateLinkage refactoring (#7517)
|\ \  
| * | 061efe9953 Migrate ARMPrivateLinkage from TR namespace to J9
```
```
$ git difftool f740577d3a bba5a43eb3 
```